lib/parser/objects.ts

```diff
@@ -1053,6 +1065,11 @@ export function* scope(startObject: ObjectBase): Generator<[ObjectBase, boolean]
         yield [packages, directlyVisible];
       }
     }
+    if (implementsIHasUseClause(current)) {
+      for (const packages of current.packageDefinitions) {
+        yield [packages, directlyVisible];
+      }
+    }
```

Comment:
```suggestion
```
This is a duplicate of lines 1062-1067 but without assigning the `directlyVisible`.

Reply:
Hups, that a artifact of the merge madness.